dbms/src/Common/OptimizedRegularExpression.inl.h

```diff
@@ -318,13 +324,13 @@ OptimizedRegularExpressionImpl<thread_safe>::OptimizedRegularExpressionImpl(cons
 
         re2 = std::make_unique<RegexType>(regexp_, options);
         if (!re2->ok())
-            throw Poco::Exception("OptimizedRegularExpression: cannot compile re2: " + regexp_ + ", error: " + re2->error());
+            throw Poco::Exception(fmt::format("OptimizedRegularExpression: cannot compile re2: {}, error: {}", regexp_, re2->error()));
 
         if (!is_no_capture)
         {
-            number_of_subpatterns = re2->NumberOfCapturingGroups();
-            if (number_of_subpatterns > MAX_SUBPATTERNS)
-                throw Poco::Exception("OptimizedRegularExpression: too many subpatterns in regexp: " + regexp_);
+            capture_num = re2->NumberOfCapturingGroups();
+            if (capture_num > MAX_CAPTURES)
+                throw Poco::Exception(fmt::format("OptimizedRegularExpression: too many subpatterns in regexp: {}", regexp_));
```

Comment:
I think we can rename all the `subpatterns` to `capture_groups`

Reply:
> I think we can rename all the `subpatterns` to `capture_groups`

okk